apps/mobile/mobile-app/components/impact/latam-impact-section.tsx

```
@@ -0,0 +1,324 @@
+} from 'lucide-react-native'
+import { MotiView } from 'moti'
+import React, { useRef, useState, useEffect } from 'react'
+import { 
+	Text as RNText, 
+	View, 
+	FlatList, 
+	Dimensions, 
+	Pressable,
+	NativeScrollEvent,
+	NativeSyntheticEvent
+} from 'react-native'
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)